src/esp/gfx/RenderTarget.cpp

```diff
@@ -116,9 +116,10 @@ struct RenderTarget::Impl {
 
   void blitRgbaToDefault() {
     framebuffer_.mapForRead(RgbaBuffer);
-    GL::AbstractFramebuffer::blit(framebuffer_, GL::defaultFramebuffer,
-                                  {{}, framebufferSize()},
-                                  GL::FramebufferBlit::Color);
+    GL::AbstractFramebuffer::blit(
+        framebuffer_, GL::defaultFramebuffer, framebuffer_.viewport(),
+        GL::defaultFramebuffer.viewport(), GL::FramebufferBlit::Color,
+        GL::FramebufferBlitFilter::Nearest);
```

Comment:
`Linear`? Or it looks *too* bad? :) Btw., how can it happen that the framebuffers have a different size? I'd assume the `framebuffer_` gets created from default FB's size and then resized as appropriate on webgl canvas / window resize (application's `viewportEvent()`).

Reply:
I can check Linear. Nearest was the default. Thanks.

About different size of frame buffers: They still have same size but the viewport can have different start and end points, so I am just taking that in consideration. For WebVR, first paint has defaultFrameBuffer's viewport as `{{0, 0}, {halfWidth, height}}`, and the second paint will have it as `{{halfWidth, 0}, {width, height}}`. So, source rectangle stays same as whatever was the viewport of the `framebuffer_` and destination rectangle would be the current viewport of the defaultFrameBuffer.

Reply:
Ah, I see :) If the size is the same, then `Nearest` makes sense, yup ... but otherwise it might look quite bad, depending on the ratio. So maybe instead add an "explanatory assert" that the sizes are equal?

Reply:
Maybe an assertion to confirm that source and destination are the same size to verify that no interpolation is happening. The interpolation method only matters if the sizes are different.